### PR TITLE
iOS: adding support to allow Music.app to play at the same time.

### PR DESCRIPTION
**Pull request by pponso1**
I added a couple lines of code in the iOS project to allow the Music.app to play while RetroArch is running. Compiled, deployed, and after listening to my jams whilst I Ocarina of Time… I have to say I can’t go back to a version of libretro without it.

I kinda sprinkled the lines of code in various places until the audio behaved in a normal manner, so I need to clean that up.

Suggestions or guidelines appreciated.

#if defined(HAVE_COCOATOUCH)
    AVAudioSession *audioSession = [AVAudioSession sharedInstance];
    [audioSession setCategory: AVAudioSessionCategoryAmbient error: nil];
#endif

### CODE REVIEW

**Comment by Alcaro** [comment]
...10000 commits?

I think I'll have to walk you through how to create a proper pull request. This one isn't really useful.

File: .gitignore [hunk 1]
```diff
@@ -9,11 +9,12 @@
 *.map
 config.log
 /retroarch
-/retroarch.*
+/retroarch.cfg
+/retroarch.exe
```
Comment (RobLoach) [inline .gitignore]: .* covers .cfg and .exe... Could ! the ones that you actually want in there. Same goes for joyconfig.

File: .project [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<projectDescription>
+	<name>RetroArch_</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```
Comment (RobLoach) [inline .project]: What's this meta data for?

File: AUTHORS [hunk 3]
```diff
@@ -1,51 +0,0 @@
...
-   - OpenPandora port
-
-Fabian Knopf -
-   - WGL multi-monitor support
-
-Gražvydas Ignotas -
-   - ARM Linux patches
-
-Saggi Mizrahi - 
-   - RetroLaunch utility
```
Comment (RobLoach) [inline AUTHORS]: Likely want to keep AUTHORS in there.

**Comment by Alcaro** [comment]
Thanks, but there's no point reviewing this PR, he made it from wrong branch. He's trying to merge master into the android-thread-input branch, which is three years old and contains nothing relevant.

**Comment by RobLoach** [comment]
After I posted those few comments, I saw you close it. Thanks! :+1:

**Pull request closed without merging**